testsuite/classlibrary/TestUGen_RTAlloc.sc

```diff
@@ -1,326 +1,322 @@
 TestUGen_RTAlloc : UnitTest {
 
-	var server;
+	classvar server;
 
-	setUp {
-		server = Server(this.class.name);
-		server.options.memSize = 1024;
+	*initClass {
+		passVerbosity = UnitTest.brief;
+	}
+
+	*setUpClass {
+		server = Server(this.name);
+		server.options.sampleRate = 48000;
 		server.options.blockSize = 64;
-		this.bootServer(server);
+		server.options.memSize = 2 ** 13; // scsynth default
+		// - tests fail with memSize < 256 (GVerb fails)
+		// - testing with memSize >= 2 ** 20 would require multiple allocations
+		//   (allocation size itself can overflow size_t for some UGens)
+		server.bootSync;
 	}
 
-	tearDown {
+	*tearDownClass {
 		server.quit.remove;
 	}
 
-	test_allocFail_clearUnit {
-		var cond = CondVar(), output = -1;
-		{ DelayN.ar(DC.ar(1), 100); }.loadToFloatArray(1 / server.sampleRate, server, {|val|
-			output = val[0];
-			cond.signalOne();
-		});
-
-		cond.waitFor(1, { output != -1 });
-		this.assertFloatEquals(output, 0.0, "a failed RTAlloc should clear unit outputs")
-	}
+	// Async helpers
 
-	test_allocFail_continueProcessing_sameNode {
-		var cond = CondVar(), output = -1;
-		{ DelayN.ar(Silent.ar, 100); DC.ar(1)}.loadToFloatArray(1 / server.sampleRate, server, {|val|
-			output = val[0];
+	awaitSynthOutput { |synthFn, runtimeSamples=(server.options.blockSize), timeout=1|
```

Comment:
Could you just remind me when this value for the runtime samples is calculated as I actually don't know!? If you change the server options at some arbitrary moment, how does this behave?

Reply:
Note that the server has to be rebooted for the server.options.blockSize to take effect.

Reply:
So it does `a = a ?? { server.options...}`.
Thanks!